qiita_db/support_files/patches/test_db_sql/92.sql

```diff
@@ -942,3 +942,13 @@ WHERE email = 'test@foo.bar';
 INSERT INTO qiita.qiita_user VALUES ('justnow@nonvalidat.ed', 5, '$2a$12$gnUi8Qg.0tvW243v889BhOBhWLIHyIJjjgaG6dxuRJkUM8nXG9Efe', 'JustNow', 'NonVeriUser', '1634 Edgemont Avenue', '303-492-1984', NULL, NULL, NULL, false, NOW(), NULL, NULL, NULL);
 INSERT INTO qiita.qiita_user VALUES ('ayearago@nonvalidat.ed', 5, '$2a$12$gnUi8Qg.0tvW243v889BhOBhWLIHyIJjjgaG6dxuRJkUM8nXG9Efe', 'Oldie', 'NonVeriUser', '172 New Lane', '102-111-1984', NULL, NULL, NULL, false, NOW() - INTERVAL '1 YEAR', NULL, NULL, NULL);
 INSERT INTO qiita.qiita_user VALUES ('3Xdays@nonvalidat.ed', 5, '$2a$12$gnUi8Qg.0tvW243v889BhOBhWLIHyIJjjgaG6dxuRJkUM8nXG9Efe', 'TooLate', 'NonVeriUser', '564 C Street', '508-492-222', NULL, NULL, NULL, false, NOW() - INTERVAL '30 DAY', NULL, NULL, NULL);
+
+-- Jul 1, 2024
+-- Inserting a human_reads_filter_method and assigning it to the raw data in prep/artifact 1
+INSERT INTO qiita.human_reads_filter_method (
+    human_reads_filter_method)
+  VALUES (
+    'The greatest human filtering method');
```

Comment:
Does 'greatest human filtering method' mean the most current, the most correct, the most aggressive filtering?

Reply:
All the above, it's a tests, it doesn't matter.